modules/integration/src/test/scala/scala/cli/integration/BspTestDefinitions.scala

```diff
@@ -149,6 +149,40 @@ abstract class BspTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  val importPprintOnlyProject = TestInputs(
+    Seq(
+      os.rel / "simple.sc" -> s"import $$ivy.`com.lihaoyi::pprint:${Constants.pprintVersion}`"
+    )
+  )
+
+  test("setup-ide should succeed for valid dependencies") {
+    importPprintOnlyProject.fromRoot { root =>
+      val p = os.proc(
+        TestUtil.cli,
+        "setup-ide",
+        ".",
+        extraOptions,
+        "--dependency",
+        s"org.scalameta::munit:${Constants.munitVersion}"
+      ).call(cwd = root, check = false)
+      expect(p.exitCode == 0)
```

Comment:
Just removing `check = false` ought to be enough, no need to check for a `0` exit code.

Reply:
Done